### PR TITLE
Women aren't allowed to be captains anymore

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
I don't do it out of any personal grudge.
Shout-out to @PrimeDSS13 for wanting to do it
stuff

Adds a gender blacklist system
Damn, we're already miles better than every other codebase already

:cl:
 * tweak: Women cannot be captains anymore.

### CODE REVIEW

**Comment by RFultra** [comment]
This is kinda cringe ngl

**Comment by Kammerjunk** [comment]
you are going to loose commit

**Comment by MadmanMartian** [comment]
wew

**Comment by ShiftyRail** [comment]
The only reason I'm not merging this is that you didn't remove the female gender, which was the objectively correct way to do it

It wouldn't even conflict with the secret repo

**Comment by PrimeDSS13** [comment]
unironically good PR, and it's true Barry is only doing it because I asked him to
I support this greatly

**Comment by Kammerjunk** [comment]
imagine taking advice from the guy who thinks SS13 is a harem
image

**Comment by PrimeDSS13** [comment]
delete this kammer that post was a joke

**Comment by Kurfursten** [comment]
Finally, the supportcode needed to lock medbay to only female characters

Too long have we suffered under the yoke of having to be surgeried by male doctors

**Comment by BomberBro** [comment]
The PR that saved veegee station.

**Comment by boy2mantwicethefam** [comment]
@ShiftyRail Maybe another time, one change at a time.
@Kurfursten It is entirely possible, yes

**Comment by Irradiation** [comment]
I wonder how long until this post gets plastered with all the associated emojis and the emoji poster gets banned for spamming.

**Comment by ghost** [comment]
Why is the obvious joke PR being put forward when the correct security PR that @PrimeDSS13 suggested is being ignored and ridiculed?

**Comment by N3X15** [comment]
> 
> 
> Why is the obvious joke PR being put forward when the correct security PR that @PrimeDSS13 suggested is being ignored and ridiculed?

Welcome to /vg/

**Comment by APCUnlocked** [comment]
Now this is based.

**Comment by ArchSunder** [comment]
BASTE

**Comment by cloutcode** [comment]
only once grays can become captain

**Comment by PrimeDSS13** [comment]
>closing this PR
bro

**Comment by PrimeDSS13** [comment]
hey this was a legit PR you can't just close it because you dislike it

**Comment by SonixApache** [comment]
thank you @Zth--

File: code/modules/mob/new_player/new_player.dm [hunk 1]
```diff
@@ -364,6 +364,10 @@
 		if(!job.species_whitelist.Find(client.prefs.species))
 			to_chat(src, alert("[rank] is not available for [client.prefs.species]."))
 			return 0
+	if(job.gender_blacklist.len)
+		if(job.gender_blacklist.Find(client.prefs.gender))
+			to_chat(src, alert("[rank] is not available for [client.prefs.species]s."))
```
Comment (SonixApache) [inline code/modules/mob/new_player/new_player.dm:369]: 🤔

Comment (boy2mantwicethefam) [reply code/modules/mob/new_player/new_player.dm:369]: will output males or females

Comment (DamianX) [reply code/modules/mob/new_player/new_player.dm:369]: will it really?

**Comment by mrflarg** [comment]
why isn't barry an admin?

**Pull request closed without merging**